sbin/shipper_utils/update_and_install_workers.py

```diff
@@ -65,13 +65,14 @@ def update_docker_images(user, host, worker, autograding_workers, autograding_co
         docker_images_obj = client.images.list()
         #print the details of the image
         get_docker_info.printDockerInfo()
-        
+        os.system('lsb_release -a')
```

Comment:
From above, we can replace this with
```suggestion
        subprocess.run(['lsb_release', '-a'], capture_output=True, check=True)
```

Note, this change does mean that if the command exits with a non-zero exit code, then an exception will be raised (instead of it being silently ignored like `os.system`), but that's probably what we want here instead of having a broken log file that we only notice when we go to parse it.